### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][JAWS/NVDA][MAS4.2.1][IntelliJ][Wipro]: NVDA and jaws are not announcing the content present in " Push Image" window sequentially

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1516]
**Repro Steps:**

1. Create a new dynamic web project.
2. Navigate to and select Project ->Azure->Push Image.
3. Push image dialog will open.
4. In Run on web App containers dialog, Press "Insert+B" to read  the entire dialog then observe the NVDA and JAWS Behavior

**Expected Result:**
NVDA and Jaws Should announce in the following manner

- Push Image  dialog  
- Name:  edit  alt+N
- Push Image: sample:sample
- Share  check box  not checked  alt+S
- Single instance only  check box  not checked  alt+I
- Invalid Data: Please specify a valid docker file.
- Artifact
- Docker File
  Docker file  edit field
- Browse  button  Shift+Enter
- Server URL
  Server URL edit field
- Username
  USer Name edit field
- Password
   Password edit field
- Image and tag
   edit field
- Artifact
  combo box  collapsed
   list items
- Before launch: Build Artifacts, Activate tool window  alt+B
 Add (Alt+Insert)  button  Add (Alt+Insert)
 Remove (Alt+Delete)  button  Remove (Alt+Delete)
 Edit (Enter)  button  Edit (Enter)
 Up (Alt+Up)  button  Up (Alt+Up)
 Down (Alt+Down)  button  Down (Alt+Down)
 separator
- list
 Build 'unnamed' artifact  1 of 1
- Show this page  check box  not checked
- Activate tool window  check box  checked
- Invalid Data: Please specify a valid docker file.
- button  Show help contents
- Run  button
- Cancel  button
- Apply  button  alt+A

**Actual Result:**
**NVDA Behavior:** NVDA is annoucing as
Push Image  dialog  
Invalid Data: Please specify a valid docker file.
Artifact
Docker File
Server URL
Username
Password
Image and tag
Container Registry
 /
Before launch: Build Artifacts, Activate tool window
   Invalid Data:  Please specify a valid docker file.  
separator
Artifact
combo box  collapsed
Docker File
edit
blank
Browse  button  Shift+Enter
Server URL
edit
blank
Username
edit
blank
Password
password edit  editable
blank
Image and tag
edit
blank
Container Registry
combo box  collapsed
 /
Before launch: Build Artifacts, Activate tool window  alt+B
separator
list
Build 'unnamed' artifact  1 of 1
Add (Alt+Insert)  button  Add (Alt+Insert)
Remove (Alt+Delete)  button  Remove (Alt+Delete)
Edit (Enter)  button  Edit (Enter)
Up (Alt+Up)  button  Up (Alt+Up)
Down (Alt+Down)  button  Down (Alt+Down)
Show this page  check box  not checked
Activate tool window  check box  checked
Name:  alt+N
Name:  edit  alt+N
Push Image: sample:sample
Share  check box  not checked  alt+S
Single instance only  check box  not checked  alt+I
button  Show help contents
filler
Run  button
Cancel  button
Apply  button  alt+A
**Jaws Behavior:**
Push Image dialog
Invalid Data: Please specify a valid docker file.Artifact Docker File Server URL Username Password Image and tag Container Registry <Server URL>/ Before launch: Build Artifacts, Activate tool window 
Show help contents Button

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jet brains.
OSBuild Version: Version 1709(Os Build 16299.192)

![push image-insert-b](https://user-images.githubusercontent.com/22956130/36150888-41857196-10eb-11e8-87f6-4468bffe94d7.jpg)
![hierarcy is incorrect for push image window](https://user-images.githubusercontent.com/22956130/36150903-4a10c928-10eb-11e8-955c-958c435d1ae9.jpg)

**Comment by Eskibear** [comment]
Content hierarchy is provided by IntelliJ run/debug configuration, out of the toolkit''s control.

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by sallyur** [comment]
As mentioned above, in our practice we cannot reproduce the issue, as NVDA cannot read content of IntelliJ IDEA at all.

- Importance of experience: it depends, varies from somewhat important to important.
- Quality of experience: not impacted. Since the Narrator is not experiencing this kind of issues, users can use the Narrator to as an alternative workaround.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749979
